js/library.js

```diff
@@ -33,6 +33,134 @@
     });
 })(wp.media.view);
 
+function createPopup() {
```

Comment:
NIT. The name is too generic; you can use something like `createPopupProBlocker`, or if you render with PHP, just `togglePopupProBlocker` to express its purpose better.